### PR TITLE
fix: additional coverage testing for data spreadsheet

### PR DESCRIPTION
**Pull request by davidmenendez**
Contributes to #2324 

it doesn't look like much, but believe me this required extensive testing and code running to get to this point. there's still a lot of coverage left untested that i'm still not 100% sure how to tackle at this point. some of which just requires more deep diving into RTL and data spreadsheet and some of which i don't think is really easily achievable because it involves mocking OS level functionality.

we should definitely look to continue to improve this incrementally. this is probably the best i can do at this point though.

### CODE REVIEW

File: packages/cloud-cognitive/src/components/DataSpreadsheet/DataSpreadsheet.js [hunk 1]
```diff
@@ -77,7 +76,7 @@ export let DataSpreadsheet = React.forwardRef(
   (
     {
       // The component props, in alphabetical order (for consistency).
-      cellSize = defaults.cellSize,
+      cellSize,
```
Comment (lee-chase) [inline packages/cloud-cognitive/src/components/DataSpreadsheet/DataSpreadsheet.js:79]: Why was the default cellSize removed?

Comment (lee-chase) [reply packages/cloud-cognitive/src/components/DataSpreadsheet/DataSpreadsheet.js]: @davidmenendez defaults.cellSize setting was also removed.

**Comment by davidmenendez** [comment]
@lee-chase changes made

**Comment by davidmenendez** [comment]
@lee-chase ^^

**Review by lee-chase — APPROVED** [review]

**Pull request merged**